src/components/ContentHighlights/HighlightStepper/tests/HighlightStepperConfirmContent.test.jsx

```diff
@@ -1,55 +1,18 @@
-import React, { useState } from 'react';
 import { screen } from '@testing-library/react';
 import '@testing-library/jest-dom/extend-expect';
-import algoliasearch from 'algoliasearch/lite';
 import { renderWithRouter } from '@edx/frontend-enterprise-utils';
-import configureMockStore from 'redux-mock-store';
-import thunk from 'redux-thunk';
-import { Provider } from 'react-redux';
 import HighlightStepperConfirmContent, { BaseReviewContentSelections, SelectedContent } from '../HighlightStepperConfirmContent';
 import {
   DEFAULT_ERROR_MESSAGE,
-  testCourseAggregation,
-  testCourseData,
 } from '../../data/constants';
-import { ContentHighlightsContext } from '../../ContentHighlightsContext';
-import { configuration } from '../../../../config';
-
-const mockStore = configureMockStore([thunk]);
-const enterpriseId = 'test-enterprise-id';
-const initialState = {
-  portalConfiguration:
-    {
-      enterpriseSlug: 'test-enterprise',
-      enterpriseId,
-    },
-};
-
-const searchClient = algoliasearch(
-  configuration.ALGOLIA.APP_ID,
-  configuration.ALGOLIA.SEARCH_API_KEY,
-);
-
-// eslint-disable-next-line react/prop-types
-const HighlightStepperConfirmContentWrapper = ({ children, currentSelectedRowIds = [] }) => {
-  const contextValue = useState({
-    stepperModal: {
-      isOpen: false,
-      highlightTitle: null,
-      titleStepValidationError: null,
-      currentSelectedRowIds,
-    },
-    contentHighlights: [],
-    searchClient,
-  });
-  return (
-    <Provider store={mockStore(initialState)}>
-      <ContentHighlightsContext.Provider value={contextValue}>
-        {children}
-      </ContentHighlightsContext.Provider>
-    </Provider>
-  );
-};
+import { TEST_ENTERPRISE_ID } from '../../../../data/tests/constants';
+import {
+  testCourseData,
+  ContentHighlightsContext,
+  initialStateValue,
+  testCourseAggregation,
+} from '../../../../data/tests/ContentHighlightsTestData';
+import 'jest-canvas-mock';
```

Comment:
[suggestion] Might recommend putting this in `setupTest.js` so we load the canvas mocks for Jest across all tests in the repo, not just this one test file (similar to how to use `jest-localstorage-mock`).